src/api/Config/types.ts

```diff
@@ -69,6 +68,7 @@ export interface IDatabaseConfig {
   username?: string;
   password?: string;
   connectionRetries?: number;
+  queryTimeout?: number;
```

Comment:
I suggest "transactionTimeout", I reflects the reality and is standard in the DB world. cc @Diachk 


```suggestion
  transactionTimeout?: number;
```